ingest/expression_files/dense_ingestor.py

```diff
@@ -66,8 +70,8 @@ def execute_ingest(self):
         # Reset csv reader to first gene row
         self.csv_file_handler = self.open_file(self.file_path)[0]
         next(self.csv_file_handler)
-        for gene_docs, data_array_documents in self.transform():
-            self.load(gene_docs, data_array_documents)
+        for documents, collection_name in self.transform():
```

Comment:
Having a `"""`-formatted docstring for the containing `execute_ingest` method here would be a nice touch.